public_suffix_list.dat

```diff
@@ -1217,7 +1217,6 @@ net.in
 org.in
 gen.in
 ind.in
-nic.in
```

Comment:
Don't you need to remove the comment about nic.in on line 1211 as well?

Reply:
The line 1211 was already commented out, that is why we did not change it. Also, the line says that nic.in is not an official eTLD .